tests/acceptance/help-test.js

```diff
@@ -140,38 +140,38 @@ describe('Acceptance: ember help', function() {
     });
 
     it('works', function() {
-      command.run(options, []);
-
-      let json = convertToJson(options.ui.output);
-      const expected = require('../fixtures/help/help.js');
+      command.validateAndRun([]).then(() => {
```

Comment:
I think we need to return this promise, or the tests aren't going to properly wait for the async...